AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -5307,36 +5317,37 @@ function Get-AzureADDomainInternal {
     .SYNOPSIS
     Get the Azure AD domains associated with this Azure AD tenant.
     .DESCRIPTION
-    This cmdlet is a wrapper around Get-AzureADDomain that is provided to future proof for adding cross-platform support, as AzureAD is not a cross-platform PowerShell module.
-    .PARAMETER Name
-    Specifies the name of a domain.
+    This cmdlet is a wrapper around Get-MgDomain that is provided to future proof for adding cross-platform support, as AzureAD is not a cross-platform PowerShell module.

```

Comment:
Same here, do we need to wrap it to get cross platform support?

Reply:
I have removed these, from MsGraph official doc, its cross platform